packages/frontend/component/src/ui/input/input.tsx

```diff
@@ -2,22 +2,28 @@ import clsx from 'clsx';
 import type {
   ChangeEvent,
   CSSProperties,
-  FocusEventHandler,
   ForwardedRef,
   InputHTMLAttributes,
   KeyboardEvent,
   KeyboardEventHandler,
   ReactNode,
 } from 'react';
-import { forwardRef, useCallback } from 'react';
+import {
+  forwardRef,
+  useCallback,
+  useEffect,
+  useLayoutEffect,
+  useRef,
+} from 'react';
 
 import { input, inputWrapper } from './style.css';
 
 export type InputProps = {
   disabled?: boolean;
   onChange?: (value: string) => void;
-  onBlur?: FocusEventHandler<HTMLInputElement>;
+  onBlur?: (ev: FocusEvent & { currentTarget: HTMLInputElement }) => void;
```

Comment:
`currentTarget` is added as parameter but not passed when called

Reply:
I just want to modify the FocusEvent here, Actually event.currentElement is always this <input />.